anchor-reference-server/src/main/java/org/stellar/anchor/reference/controller/RateController.java

```diff
@@ -12,9 +12,11 @@
 @RestController
 public class RateController {
   private final RateService rateService;
+  private final Gson gson;
 
-  public RateController(AppSettings appSettings, RateService rateService) {
+  public RateController(AppSettings appSettings, RateService rateService, Gson gson) {
```

Comment:
I am not sure if it is too much to autowired it? 😄  

Should we use GsonUtils.builder().create()?

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/pull/158/commits/923fdb5b54648c99affd3eda00fd458c792a217f